roles/migrationcontroller/templates/controller.yml.j2

```diff
@@ -67,6 +68,7 @@ spec:
         | join(mig_pod_limit | string)
         | join(mig_namespace_limit | string)
         | join(discovery_volume_path | string)
+        | join(mig_failure_rollback | string)
```

Comment:
@jmontleon can you ack this change that involved configHash additions?

Reply:
Looks correct to me here as well.